gui/wxpython/modules/import_export.py

```diff
@@ -559,6 +563,8 @@ def __init__(self, parent, giface, link=False):
 
         self.layersData = []
 
+        self.showModulesDisplayInfo = Signal('OgrImportDialog.showModulesDisplayInfo')
```

Comment:
It looks like both import dialogs have the same signal and both already inherit from the same parent class. It seems that the signal should be part of the interface of the parent class, i.e., any import dialog. There seems to be some code duplication in OnCmdDone, but the signal can be emitted in the child classes, that's fine.

(No harm done if the DXF dialog has that part not implemented.)

Reply:
Signal is now created in the ImportDialog.